python/lsst/ip/isr/calibType.py

```diff
@@ -333,12 +336,20 @@ def readText(cls, filename, **kwargs):
         """
         if filename.endswith((".ecsv", ".ECSV")):
             data = Table.read(filename, format='ascii.ecsv')
-            return cls.fromTable([data], **kwargs)
+            if 'CALIBCLS' in data.meta:
+                storedClass = doImport(data.meta['CALIBCLS'])
+                return storedClass.fromTable([data], **kwargs)
+            else:
+                return cls.fromTable([data], **kwargs)
```

Comment:
Quick note that I don't like duplicating the business logic here (and elsewhere). I'd prefer. the if-clause be determining the correct class. So something like:
```python
calib_cls = cls
if "CALIBCLS" in data.meta:
    calib_class = doImport(data.meta["CALIBCLS"])
return calib_class.fromTable([data], **kwargs)
```

PS Single vs double quote consistency.